tv_apps/fling-player/style/fling_player.css

```diff
@@ -91,73 +81,84 @@ article, section, video {
 }
 
 #time-progess {
-  width: 80%;
-  height: 4rem;
-  left: 10%;
+  height: 8rem;
+  left: 19.2rem;
```

Comment:
But since the visual spec didn't define the width of time digits, it's fine to me. But it may be better to double confirm with visual.